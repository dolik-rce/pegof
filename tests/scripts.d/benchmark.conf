input scripts.d/scripts.peg
output scripts.d/benchmark.tmp
header never
optimize inline,remove-group
benchmark scripts.d/benchmark.sh
