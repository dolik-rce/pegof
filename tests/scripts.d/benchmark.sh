#!/usr/bin/env bash

set -e

case "$1" in
setup)
    echo "Setup called: $*"
    ;;
benchmark)
    echo "Benchmark called: $*"
    ;;
teardown)
    echo "Teardown called: $*"
    ;;
esac
