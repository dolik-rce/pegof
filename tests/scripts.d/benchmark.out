Setup called: setup /tmp/pegof_<random>/output
Teardown called: teardown /tmp/pegof_<random>/output
Setup called: setup /tmp/pegof_<random>/output
Teardown called: teardown /tmp/pegof_<random>/output
         |      lines |      bytes |      rules |      terms |   duration |     memory
---------+------------+------------+------------+------------+------------+-----------
input    |   <number> |   <number> |   <number> |   <number> |   <number> |   <number>
output   |   <number> |   <number> |   <number> |   <number> |   <number> |   <number>
output % |  <number>% |  <number>% |  <number>% |  <number>% |  <number>% |  <number>%
